### PR TITLE
[Documentation] Concepts: Emails

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes
| New docs?    | no
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

File: docs/book/emails.rst [hunk 1]
```diff
@@ -7,73 +7,120 @@ E-Mails
...
 To understand how e-mail sending works internally, please refer to :doc:`SyliusMailerBundle documentation </bundles/SyliusMailerBundle/index>`.
 
-User Confirmation E-Mail
-------------------------
+User Confirmation
+-----------------
 
-Every time new customer registers via registration form or checkout, ``user_confirmation`` e-mail is sent to him.
-The default template is
+Every time a new customer registers via the registration form, a user confirmation e-mail is sent to them.
```
Comment (GSadee) [inline docs/book/emails.rst]: ...a new customer (...) is sent to **them**...
I am not sure if it is incorrect, but looks strange to me

File: docs/book/emails.rst [hunk 2]
```diff
@@ -7,73 +7,120 @@ E-Mails
...
 You also have the following parameters available:
 
-user
-    Instance of the user model
+* ``user``: Instance of the user model
+
+Email Verification
+------------------
+
+When the a new customer registers via the registration form, besides the User Confirmation an Email Verification is sent.
```
Comment (GSadee) [inline docs/book/emails.rst]: Remove **the**

File: docs/book/emails.rst (hunk 1 above)
Comment (TheMadeleine) [inline docs/book/emails.rst]: It's gender neutral :)

Comment (bretto36) [inline docs/book/emails.rst]: Every time a customer registers via the registration form, a user confirmation e-mail is sent to them.

File: docs/book/emails.rst [hunk 3]
```diff
@@ -7,73 +7,120 @@ E-Mails
...
-    Instance of the user order
-order.user
-    Customer
-order.shippingAddress
-    Shipping address
-order.billingAddress
-    Billing address
-order.items
-    Collection of order items
+* ``order``: Instance of the order, with all its data
```
Comment (bretto36) [inline docs/book/emails.rst]: it's

File: docs/book/emails.rst [hunk 4]
```diff
@@ -7,73 +7,120 @@ E-Mails
...
-    Shipping address
-order.billingAddress
-    Billing address
-order.items
-    Collection of order items
-
-Final Thoughts
---------------
+* ``comment``: Comment instance
+* ``order``: Instance of the order, with all its data
```
Comment (bretto36) [inline docs/book/emails.rst]: it's

File: docs/book/emails.rst [hunk 5]
```diff
@@ -7,73 +7,120 @@ E-Mails
...
+This e-mail is sent when the order's shipping process has started.
+
+**Code**: ``shipment_confirmation``
+
+**The default template**: ``SyliusCoreBundle:Email:shipmentConfirmation.html.twig``
+
+You have the following parameters available:
+
+* ``shipment``: Shipment instance
+* ``order``: Instance of the order, with all its data
```
Comment (bretto36) [inline docs/book/emails.rst]: it's

File: docs/book/emails.rst (hunk 3 above)
Comment (steffenbrem) [inline docs/book/emails.rst]: @bretto36 its is correct in this case, since it is used in it's possessive form. It's is a contraction of `it is`.

Comment (bretto36) [inline docs/book/emails.rst]: well i feel like an idiot haha. Did you use it incorrectly to make me feel better, or to confuse me

"used in it's possessive form."

Comment (steffenbrem) [inline docs/book/emails.rst]: hahaha wow, i am not a native English speaker myself so I slipped up too there 😋. So easy to make those.

**Comment by pjedrzejewski** [comment]
Thank you Magda!

**Pull request merged**